osbuild.spec

```diff
@@ -9,7 +9,7 @@ Version:        148
 %global         pkgdir %{_prefix}/lib/%{pypi_name}
 
 Name:           %{pypi_name}
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Please don't bump the release number.

We have bi-weekly releases, so the version in the SPEC is the one that will be released next. We never bump release numbers in the SPEC file.

Reply:
Note that this is purely an upstream thing.

Reply:
Makes sense. I’ll adjust this and force-push.